src/main/kotlin/org/jitsi/nlj/RtpSenderImpl.kt

```diff
@@ -96,23 +95,13 @@ class RtpSenderImpl(
 
     private val outputPipelineTerminationNode = object : ConsumerNode("Output pipeline termination node") {
         override fun consume(packetInfo: PacketInfo) {
-            if (packetInfo.timeline.totalDelay() > Duration.ofMillis(100)) {
-                logger.cerror { "Packet took >100ms to get through bridge:\n${packetInfo.timeline}" }
-            }
+            packetDelayStats.addPacket(packetInfo)
```

Comment:
I think we should move this to `DtlsTransport`, because there is a whole extra queue (running in another thread) before the packet hits the socket.